code/modules/projectiles/projectile/beams.dm

```diff
@@ -81,6 +85,26 @@
 	muzzle_type = /obj/effect/projectile/muzzle/disabler
 	impact_type = /obj/effect/projectile/impact/disabler
 
+// Plasmagun Start
+
+/obj/item/projectile/beam/laser/blue
+	name = "energy beam"
+	icon_state = "omnilaser"
+	damage = 20
+	damage_type = BURN
+	flag = "energy"
+	tracer_type = /obj/effect/projectile/tracer/laser/blue
+	muzzle_type = /obj/effect/projectile/muzzle/laser/blue
+	impact_type = /obj/effect/projectile/impact/laser/blue
+
+/obj/item/projectile/beam/laser/blue/triple
+	icon_state = "omnilaser"
+	damage = 6 // Does less damage due to cheaper energy cost.
+	eyeblur = 1
+	stamina = 5 // Bonus! It's function
```

Comment:
18+15 = 33 not 18

Reply:
I forgot to finish the comment.
Does the 18 burn damage contribute to deciding when you get stam-crit?
This doesn't do 33 burn damage, it does 18 if all shots connect and 15 stamina.

Reply:
i know it isn't obtainable outside of admin spawn but omitting information in changelog is shitty. this is the same reason why we had instadeath for ipcs for half a year on emp because people don't know how to not lie through omission

Reply:
Assuming they're lying on purpose instead of just missing something is a really shitty thing to do with something this big. Maintainers are supposed to review shit for a reason.

Reply:
Urh, this isn't anywhere near that bad. I forgot, I added it in. I forgot to mention it in the word docx that I was recording changes in because I was half asleep whilst doing 70% of this PR.
Thanks for pointing it out

Reply:
"never attribute to malice that which can be adequately explained by stupidity"
-gilette man (idk i dont use razors)

Reply:
No off topic comments please